### ISSUE TITLE
Using a feed aggregator gets flooded with errata commits

### ISSUE THREAD

**Issue by vjackson725** [frankmcsherry/blog#10]
Hi,

The way you're doing your blog now makes it hard to follow if you're using a feed aggregator (like I am) to follow the posts, as they get lost in all the formatting/correction commits.

Although it's not a terribly clean solution, it would help if you kept a separate branch and `git merge --squash`-ed the changes every time you finished a new blog post.

**Comment by frankmcsherry** [comment]
That's a good point, thanks. I'll try this from now on; with the next post, I'll drop in on this issue and mention the new branch (or fail to, at which point this is the right place to remind me at which point I'll apologize and do it :)).

**Comment by frankmcsherry** [comment]
There is now a `squashed` branch, which I will `git merge --squash` into when posts land (perhaps shortly after, to absorb any immediate edits). Give a holler if I screw up the intended behavior! :)
